src/SingleBookingPage/BookingDetail.js

```diff
@@ -93,6 +93,7 @@ class BookingDetail extends React.Component<Props> {
 
   render() {
     const { booking } = this.props;
+    const eTicketLink = idx(booking, _ => _.assets.ticketUrl) || null;
```

Comment:
Not sure if you need to specify `null` here

Reply:
no, is not strictly necessary, changed.